pkg/rtc/mediatrack.go

```diff
@@ -119,6 +119,10 @@ func NewMediaTrack(track *webrtc.TrackRemote, params MediaTrackParams) *MediaTra
 	t.AddOnClose(t.closeChan)
 	go t.updateStats()
 
+	time.AfterFunc(30*time.Second, func() {
```

Comment:
Yeah, will move it. Making some changes to start this when a receiver is created. Also, adding a `Stop` on the timer just in case the track closes before 30 seconds.